core/server/src/main/java/alluxio/master/file/FileSystemMaster.java

```diff
@@ -421,10 +424,32 @@ public void start(boolean isLeader) throws IOException {
       mLostFilesDetectionService = getExecutorService().submit(new HeartbeatThread(
           HeartbeatContext.MASTER_LOST_FILES_DETECTION, new LostFilesDetectionHeartbeatExecutor(),
           Configuration.getInt(PropertyKey.MASTER_HEARTBEAT_INTERVAL_MS)));
+      mStartupConsistencyCheck = getExecutorService().submit(new Callable<List<AlluxioURI>>() {
```

Comment:
When will this check start running? Before or after the master recovers from the journal?

Is the master serving during the check? This means only read only operations would be possible during the check?

Also, how does this interact with standby masters becoming leader masters?